test/groovy/DockerExecuteOnKubernetesTest.groovy

```diff
@@ -809,6 +820,20 @@ class DockerExecuteOnKubernetesTest extends BasePiperTest {
             hasEntry('excludes', 'container/exclude.test'))))
     }
 
+    @Test
+    void testUnstash() {
+        stepRule.step.dockerExecuteOnKubernetes(
+            script: nullScript,
+            juStabUtils: utilsMock,
+            dockerImage: 'maven:3.5-jdk-8-alpine',
+        ) {
+            bodyExecuted = true
```

Comment:
should be check that body has been executed? If not I suggest to remove it since this is confusing.

Reply:
https://github.com/SAP/jenkins-library/pull/4381/commits/c0ca2cdbe6d34db9e5b7775085ad5d7e832d4a6b